timesketch/__init__.py

```diff
@@ -11,3 +11,87 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Entry point for the application."""
+
+import os
+
+from flask import Flask
+from flask_login import LoginManager
+from flask_restful import Api
+from flask_wtf import CsrfProtect
+from timesketch.api.v1.resources import ExploreResource
+from timesketch.api.v1.resources import EventResource
+from timesketch.api.v1.resources import EventAnnotationResource
+from timesketch.api.v1.resources import SketchResource
+from timesketch.api.v1.resources import SketchListResource
+from timesketch.api.v1.resources import ViewResource
+from timesketch.api.v1.resources import ViewListResource
+from timesketch.models import configure_engine
+from timesketch.models import init_db
+from timesketch.models.sketch import Sketch
+from timesketch.models.user import User
+from timesketch.ui.views.home import home_views
+from timesketch.ui.views.sketch import sketch_views
+from timesketch.ui.views.user import user_views
+
+
+def create_app(config=None):
+    """Create the Flask app instance that is used throughout the application."""
+    # Setup the Flask app and load the config.
+    app = Flask(
+        __name__, template_folder='ui/templates', static_folder='ui/static')
+    if not config or isinstance(config, str):
```

Comment:
Why not:
if not config:
    config = '/etc/timesketch.conf'
if isinstance(config, basestring):
  ...